### ISSUE TITLE
NetSynth Bug on Linux

### ISSUE THREAD

**Issue by scvannost** [CIDARLAB/cello#38]
I'm trying to run Cello on Ubuntu 16.04LTS, and netsynth.Utilities.getFileContentAsStringList() is throwing a FileNotFoundException on results/user/demo001/netSynth/abcOutput.bench. This happens on both the local web app and using the API. It's reporting the error as "Error in abstract circuit." from line 235 in DNAcompiler.run().

Compiling using Maven; JDK7

**Comment by tim-tx** [comment]
Can you confirm that you created a results folder mirroring the name of the source directory, as documented in [INSTALL.md](https://github.com/CIDARLAB/cello/blob/develop/INSTALL.md)?

**Comment by scvannost** [comment]
Yes I did. It's creating the netSynth folder in the results as well.

**Comment by emernic** [comment]
I have encountered this issue as well, also on Ubuntu.
